### PR TITLE
i18n(hi): update `404.md`

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description

- updates 404.md for Hindi language

### CODE REVIEW

File: docs/src/content/docs/hi/404.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Not found
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/404.md:2]: Thanks for your contribution 🙌 

Small question: the title can be safely translated, would it make sense or improve the experience to translate it in Hindi too?

Comment (at-the-vr) [reply docs/src/content/docs/hi/404.md]: of course, I completely omitted title at first but with latest commit that should be resolved)

File: docs/src/content/docs/hi/404.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: पेज नहीं मिला
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: <strong>Houston, हमें एक समस्या है।</strong> हमें वह पेज नहीं मिला.<br> URL जांचें या खोज बार का उपयोग करने का प्रयास करें।
+  actions:
+    - text: अवतरण पेज पर वापस आये
+      icon: right-arrow
+      link: /
```
Comment (delucis) [inline docs/src/content/docs/hi/404.md:11]: Looks great — thank you @at-the-vr!

One last thing, the link should point to the Hindi homepage here:

```suggestion
      link: /hi/
```

**Review by delucis — APPROVED** [review]
Thanks for your patience on this one @at-the-vr! Appreciate the contribution 💖

**Pull request merged**